dist/index.js

```diff
@@ -1414,6 +1370,36 @@ async function run() {
       containerDef.logConfiguration.options["awslogs-region"] = awslogsRegion;
     }
 
+    if (logConfigOptions) {
```

Comment:
Added these two new sections, allowing us to override log configuration options.